src/stdune/path.ml

```diff
@@ -630,6 +630,20 @@ module Kind = struct
     | External x -> External (External.relative x (Local.to_string y))
 end
 
+let _chmod ~mode ?(op = `Set) path =
```

Comment:
underscore names are not great because they are ignored by the unused value check. It's better to use something like `chmod_generic`